### ISSUE TITLE
No module named encoding.datasets

### ISSUE THREAD

**Issue by Rafrin** [zhanghang1989/PyTorch-Encoding#65]
Got this error from the line 19 of test.py. (experiments/segmentation/ folder)

**Comment by zhanghang1989** [comment]
Have you install the package yet?

**Comment by Rafrin** [comment]
git clone https://github.com/zhanghang1989/PyTorch-Encoding && cd PyTorch-Encoding

you mean cloning the git? yes I did

**Comment by zhanghang1989** [comment]
python setup.py install

**Comment by Rafrin** [comment]
1. git clone https://github.com/zhanghang1989/PyTorch-Encoding && cd PyTorch-Encoding
2. python setup.py install
3. cd PyTorch-Encoding/experiments/recognition
bash model/download_models.sh
4. python main.py --dataset cifar10 --model encnetdrop --widen 8 --ncodes 32 --resume model/encnet_cifar.pth.tar --eval
5. python test.py --dataset PContext --model-zoo Encnet_ResNet50_PContext --eval

these are the steps I am going through and I am getting the error.

**Comment by zhanghang1989** [comment]
It is in the `experiments/segmentation` folder

**Comment by Rafrin** [comment]
yes, I cd to that experiment/segmentation folder and did the step 5

**Comment by zhanghang1989** [comment]
I couldn't reproduce your error on my local machine. The `encoding.datasets` source code in at https://github.com/zhanghang1989/PyTorch-Encoding/tree/master/encoding/datasets

Can you attach the whole error message if you still have the problem.

**Comment by Rafrin** [comment]
![untitled](https://user-images.githubusercontent.com/31631990/41170733-9c836398-6b1b-11e8-80c0-ca5ebf6a9b0f.png)

**Comment by Rafrin** [comment]
yes. if you look into the error message i tried with python 3

**Comment by zhanghang1989** [comment]
This looks wired. I will take a look. Thanks for reporting it.

**Comment by Rafrin** [comment]
Thank you

**Comment by zhanghang1989** [comment]
Hi @Rafrin,  can you try to uninstall pervious installed versions first. Run:
```
pip uninstall encoding
pip uninstall torch-encoding
```
until you get an error `import encoding`, then reinstall this package.

**Comment by Rafrin** [comment]
Hi, @zhanghang1989 , I am trying to follow your instructions. I don't have the authority to uninstall packages in GPU server as other students are using it. I forwarded the issue to the person in charge. It will take some time. I will let you know if I am still getting the error.

**Comment by zhanghang1989** [comment]
do you still have the problem now?

**Comment by Rafrin** [comment]
I couldn't solve the problem yet. We are trying to get over it.
